src/features/snap/components/ReportSnap.tsx

```diff
@@ -10,13 +10,13 @@ import useToastMsg from '../../../hooks/useToastMsg';
 
 type ReportSnapProps = {
   address: Hex;
-  snapId: string;
+  versionChecksum: string;
   snapName: string;
 };
 
 export const ReportSnap: FunctionComponent<ReportSnapProps> = ({
   address,
-  snapId,
+  versionChecksum,
```

Comment:
nic:
apply to all if possible
```suggestion
  snapChecksum,
```